components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add-finish-ajaxprocessor.jsp

```diff
@@ -106,7 +106,15 @@
     String spName = request.getParameter("application");
     boolean isError = false;
     OAuthConsumerAppDTO consumerApp = null;
-    
+
+    String idTokenAudienceEnabled = "enableIdTokenAudienceRestriction";
+    String idTokenAudienceCount = "idTokenAudiencePropertyCounter";
+    String idTokenAudiencesProperty = "idTokenAudiencePropertyName";
+
+    String accessTokenAudienceEnabled = "enableAccessTokenAudienceRestriction";
+    String accessTokenAudienceCount = "accessTokenAudiencePropertyCounter";
+    String accessTokenAudiencesProperty = "accessTokenAudiencePropertyName";
```

Comment:
move these constants to OAuthUIConstants class

Reply:
resolved with 08954d00bccda5645fa04905eca48d4cc7ae5f01